lib/environment/class-environment.php

```diff
@@ -9,35 +9,39 @@ class Environment {
 	 */
 	const VAR_PREFIX = 'VIP_ENV_VAR';
 
+    /**
+     * Checks existence of custom environment variable values based on the supplied key.
+     *
+     * @param string $key The name of the environment variable.
+     * @return bool
+     */
+    public static function has_var( string $key ): bool {
+        return defined( self::calculate_env_const_name( $key ) );
+    }
+
 	/**
-	 * Attempts to return custom environment variable values based on the supplied key
+	 * Attempts to return custom environment variable values based on the supplied key.
 	 *
 	 * @param string $key The name of the environment variable.
-	 * @param string $default_value The value to return if no environment variable matches the key
-	 * @return string
+	 * @param mixed $default_value The value to return if no environment variable matches the key
+	 * @return mixed
 	 */
 	public static function get_var( string $key, $default_value = '' ) {
-		$key       = strtoupper( $key );
-		$env_const = sprintf( '%s_%s', self::VAR_PREFIX, $key );
-
-		if ( defined( $env_const ) ) {
-			return constant( $env_const );
-		}
-
-		// The call was not able to retrieve an env variable
-		// log this as an E_USER_NOTICE for debugging
-		trigger_error( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
-			sprintf(
-				'The ENV variable: "%s" did not exist. %s::get_var returned the default value',
-				$key, // phpcs:ignore WordPress.Security.EscapeOutput
-				__CLASS__
-			),
-			\E_USER_NOTICE
-		);
+		$env_const = self::calculate_env_const_name( $key );
 
-		return $default_value;
+		return defined( $env_const ) ? constant( $env_const ) : $default_value;
```

Comment:
What do you think about simplifying it more to just:

```suggestion
		return self::has_var( $env_const ) ? constant( $env_const ) : $default_value;
```

Reply:
Ah, nevermind. Then `calculate_env_const_name()` would be called redundantly.

Reply:
Yep, that's what I meant in my previous comment when I said _"the second problem could be solved by duplicating the code of has_var() in get_var()."_ :)